Library/Koala/KoalaTests.swift

```diff
@@ -356,7 +356,7 @@ final class KoalaTests: TestCase {
     XCTAssertNil(properties["discover_staff_picks"])
     XCTAssertNil(properties["discover_starred"])
     XCTAssertNil(properties["discover_term"])
-    XCTAssertEqual(false, properties["discover_everything"] as? Bool)
+    XCTAssertEqual(true, properties["discover_everything"] as? Bool)
```

Comment:
This test seems unrelated, just curious why the boolean flipped?

Reply:
The value `true`  means that the array of "discovery properties"  is empty in that context. The value was `false` because we adding the `current_variants` property  to the array, but now that we moved the property to another place, the array is empty again (in this test).